src/site/content/en/blog/storage-for-the-web/index.md

```diff
@@ -159,7 +163,14 @@ of stored cross origin resources.
 During development, you can use your browser's DevTools to inspect the
 different storage types, and easily clear all stored data.
 
-<img class="w-screenshot w-screenshot-filled" alt="Storage test tool." src="storage-test-tool.png">
+A new feature was added in Chrome 88 that lets you override the storage quota
+site in the Storage Pane. This feature gives you the ability to simulate
```

Comment:
Override the storage quota site? Do you mean override the site's storage quota?

Reply:
Yep, fixed